server/api/router.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import express from 'express';
+import { createProxyServer } from 'http-proxy';
+import { UIServerModule } from 'types';
+
+import {
+  proxyErrorHandler,
+  proxyCompleteHandler,
+  proxyStartHandler,
+} from './controller';
+
+const moduleName = 'api';
+
+export const ApiModule: UIServerModule = {
+  moduleName,
+  addModule: (logGenerator, authFn, serverConfig) => {
+    const { entry } = logGenerator(moduleName);
+    const { proxy } = serverConfig;
+    const { debug, exit } = entry('addModule', proxy);
+
+    const { cert, minTLS } = proxy.transport;
+
+    const proxyConfig = {
+      target: `${cert ? 'https' : 'http'}://${proxy.hostname}:${proxy.port}`,
+      ca: cert,
+      minVersion: minTLS,
+      changeOrigin: true,
+      secure: cert ? true : false,
+    };
+
+    debug(`api proxy configuration`, JSON.stringify(proxyConfig));
```

Comment:
should this debug be gone?

Reply:
Debug are intentional debug level statements. When the logging is delivered, they will only fire when that level is set